lib/ffprobe.ex

```diff
@@ -14,11 +14,17 @@ defmodule FFprobe do
 
   @doc """
   Get the duration in seconds, as a float.
-  If no duration (e.g., a still image), returns `:no_duration`
+  If no duration (e.g., a still image), returns `:no_duration`.
+  If the file does not exist, returns { :error, :no_such_file } 
   """
   @spec duration(binary | format_map) :: float | :no_duration
```

Comment:
Would be nice to update the typespecs

Reply:
yes, sure I will.

Reply:
Oops I made a mistake, I forgot add the  { :error, :no_such_file }  to the spec. If there is anything more, I will do that too in a commit. Otherwise I will send a last commit with the correct spec.

Reply:
Everything else, other than the typespec you mentioned, looks pretty good after your updated PR. Thank you very much!